js/design.js

```diff
@@ -1,20 +1,16 @@
 'use strict';
-
-var cardContainer = document.querySelector('.viewfinder');
-var radiosGroup = document.querySelectorAll('.radiosGroup');
```

Comment:
Qué habéis hecho con esa variable?? :)

Reply:
Para ponerlo en ES6, no se necesita esa variable, en realidad en ES5 tampoco es necesaria con la variable radios es suficiente.

Reply:
Ok! Aprobado! ;)